react/features/base/lastn/middleware.js

```diff
@@ -71,11 +71,13 @@ function _updateLastN({ getState }) {
     const defaultLastN = typeof config.channelLastN === 'undefined' ? -1 : config.channelLastN;
     let lastN = defaultLastN;
 
-    // Apply last N limit based on the # of participants
-    const limitedLastN = limitLastN(participantCount, lastNLimits);
+    if (lastNLimits) {
```

Comment:
unrelated?

Reply:
Yeah, just a quick thing while Pawel pushes the proper fix in a PR :-)